server/routes/games.js

```diff
@@ -1,10 +1,10 @@
+const aqp = require('api-query-params');
 const express = require('express')
 const { isLoggedIn } = require('../middlewares')
 const Game = require('../models/Game')
-const Like = require('../models/Like')
 const router = express.Router()
 
-const populatable_virtuals = 'ratings mechanic_names category_names likes';
+const populatable_virtuals = 'reviews ratings categories mechanic_names category_names likes';

```

Comment:
Probably don't need `category_names` if we change the `Game` model to what it is now.  Just not sure how we can tie BGA Categories to games.